docs/core/lmdb_dataset_creation.md

```diff
@@ -0,0 +1,251 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Making LMDB Datasets (original format)
+
+Storing your data in an LMDB ensures very fast random read speeds for the fastest supported throughput. This is the recommended option for the majority of OCP use cases. For more information about writing your data to an LMDB, please see the [LMDB Dataset Tutorial](https://github.com/Open-Catalyst-Project/ocp/blob/main/tutorials/lmdb_dataset_creation.ipynb).
+
+This notebook provides an overview of how to create LMDB datasets to be used with the OCP repo. This tutorial is intended for those who wish to use OCP to train on their own datasets. Those interested in just using OCP data need not worry about these steps as they've been automated as part of the download script: https://github.com/Open-Catalyst-Project/ocp/blob/master/scripts/download_data.py.
+
+```{code-cell} ipython3
+from ocpmodels.preprocessing import AtomsToGraphs
+from ocpmodels.datasets import SinglePointLmdbDataset, TrajectoryLmdbDataset
+import ase.io
+from ase.build import bulk
+from ase.build import fcc100, add_adsorbate, molecule
+from ase.constraints import FixAtoms
+from ase.calculators.emt import EMT
+from ase.optimize import BFGS
+import matplotlib.pyplot as plt
+import lmdb
+import pickle
+from tqdm import tqdm
+import torch
+import os
+```
+
+## Generate toy dataset: Relaxation of CO on Cu
+
+```{code-cell} ipython3
+adslab = fcc100("Cu", size=(2, 2, 3))
+ads = molecule("CO")
+add_adsorbate(adslab, ads, 3, offset=(1, 1))
+cons = FixAtoms(indices=[atom.index for atom in adslab if (atom.tag == 3)])
+adslab.set_constraint(cons)
+adslab.center(vacuum=13.0, axis=2)
+adslab.set_pbc(True)
+adslab.set_calculator(EMT())
+dyn = BFGS(adslab, trajectory="CuCO_adslab.traj", logfile=None)
+dyn.run(fmax=0, steps=1000)
+```
+
+```{code-cell} ipython3
+raw_data = ase.io.read("CuCO_adslab.traj", ":")
+len(raw_data)
+```
+
+## Initial Structure to Relaxed Energy/Structure (IS2RE/IS2RS) LMDBs
+
+IS2RE/IS2RS LMDBs utilize the SinglePointLmdb dataset. This dataset expects the data to be contained in a SINGLE LMDB file. In addition to the attributes defined by AtomsToGraph, the following attributes must be added for the IS2RE/IS2RS tasks:
```

Comment:
This is quite outdated, both tasks use the same dataset - LmdbDataset. The old ones still exist for backwards compatibility, we should update the docs to have the new one, is less confusing.